.chloggen/release-ack-extension.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: new_component
```
It might make sense to make this `new_component` to show that it's now available for use.